app/angular/src/client/preview/types.ts

```diff
@@ -28,4 +28,5 @@ export interface StoryFnAngularReturnType {
   moduleMetadata?: NgModuleMetadata;
   template?: string;
   styles?: string[];
+  hasCustomTemplate?: boolean;
```

Comment:
Isn't there a way to set this flag in the story context? what do you think @shilman ?

Reply:
Any similar stuff being done? I could have a look, if I know where to :)